insights/parsers/tomcat_server_log.py

```diff
@@ -0,0 +1,41 @@
+"""
+TomcatServerlog file in ``/usr/share/tomcat/logs/catalina*.log`` or ``/usr/share/tomcat6/logs/catalina*.log``
+=============================================================================================================
+    .. note::
+        On RHEL6, the default tomcat server log file is under /usr/share/tomcat6/logs/.
+        On RHEL7, the default tomcat server log file is under /usr/share/tomcat/logs/.
+
+    Sample log lines::
+
+        INFO: Command line argument: -Djava.io.tmpdir=/var/cache/tomcat/temp
+        Nov 28, 2017 2:11:22 PM org.apache.catalina.startup.VersionLoggerListener log
+        INFO: Command line argument: -Djava.util.logging.config.file=/usr/share/tomcat/conf/logging.properties
+        Nov 28, 2017 2:11:22 PM org.apache.catalina.startup.VersionLoggerListener log
+        INFO: Command line argument: -Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager
+        Nov 28, 2017 2:11:22 PM org.apache.catalina.core.AprLifecycleListener lifecycleEvent
+        INFO: The APR based Apache Tomcat Native library which allows optimal performance in production environments was not found on the java.library.path: /usr/java/packages/lib/amd64:/usr/lib64:/lib64:/lib:/usr/lib
+        Nov 28, 2017 2:11:22 PM org.apache.coyote.AbstractProtocol init
+        INFO: Initializing ProtocolHandler ["http-bio-18080"]
+        Nov 28, 2017 2:11:22 PM org.apache.coyote.AbstractProtocol init
+        SEVERE: Failed to initialize end point associated with ProtocolHandler ["http-bio-18080"]
+
+
+    Examples:
+        >>> TomcatServerLog.filters.append('Failed to initialize')
+        >>> logs = shared[TomcatServerLog]
+        >>> error_msgs = logs.get('Failed to initialize')
+        >>> error_msgs[0]['raw_message']
+        >>> 'SEVERE: Failed to initialize end point associated with ProtocolHandler ["http-bio-18080"]'
+"""
+
+
+from .. import LogFileOutput, parser
+from insights.specs import tomcat_server_log
+
+
+@parser(tomcat_server_log)
+class TomcatServerLog(LogFileOutput):
+    """
+    Read the default tomcat server log file.
+    """
```

Comment:
The default `time_format` of the base class `LogOutFile` is `'%Y-%m-%d %H:%M:%S'`, You should define it for TomcatServerLog since it's different with the default one.